javascript/elements/updates_for_element.js

```diff
@@ -73,6 +73,35 @@ export default class UpdatesForElement extends SubscribingElement {
       }
 
       template.innerHTML = String(html[url(blocks[i])]).trim()
+
+      const reloadingTurboFrames = [
+        ...template.content.querySelectorAll(
+          'turbo-frame[src]:not([loading="lazy"])'
+        )
+      ]
+
+      await Promise.all(
+        reloadingTurboFrames.map(frame => {
+          return new Promise(async resolve => {
+            const frameResponse = await fetch(frame.getAttribute('src'), {
```

Comment:
It seems like this `fetch` should have the ability to report an error, similar to https://github.com/stimulusreflex/cable_ready/blob/d3e2efcffaa0646077e317cd9b6f4c3b27b45524/javascript/elements/updates_for_element.js#L70